style.css

```diff
@@ -0,0 +1,253 @@
+/* CONTAINER - it's a grid*/
+
+#container {
+    position: relative;
+    background: linear-gradient();
+    padding: 0px;
+    margin: 0px;
+    height: fit-content;
+    display: grid;
+    grid-template-columns: 2fr 2fr 2fr 2fr;
+    grid-template-rows: .5fr 1fr 1fr 1fr;
+    grid-gap: 1px;
+    grid-template-areas: 
+    "header header header header"
+    "features features features features"
+    "main main main sidebar"
+    "main main main sidebar"
+    "footer footer footer footer";
+}
+
+
+/* Header sections - handling text of the banner*/
+.header {
+    grid-area: header;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100px;
+}
+.headrln {
+    width: 65vw
+}
+h1 {
+   color: orangered; 
+   text-shadow: 2px 2px 2px darkgray;
+}
+.subhead {
+    font-size: 13px;
+    text-align: center;
+}
+
+
+/* this is the subhead section*/
+.features {
+    grid-area: features;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin: 10px;
+    /* background-color: rgba(255, 160, 122, 0.171); */
+    height: 60vh
+}
+.imgbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.img {
+    width: 25vw;
+    height: auto;
+    padding: 20px;
+}
+
+
+.banner {
+    text-align: center;
+    background-color: rgba(255, 102, 0, 0.288);
+    background-position-y: center;
+    margin-top: 60px;
+    margin-bottom: 60px;
+    height: 10vh;
+}
+
+.featsect {
+    display: flex;
+    flex-direction: column;
+    align-items: space-around;
+}
+
+button {
+    height: 3vw;
+    align-self: center;
+    margin: 5px;
+    text-align: center;
+    display: inline-block;   
+}
+.button {
+    height: 3vw;
+    align-self: center;
+    margin: 5px;
+    text-align: center;
+    display: inline-block;   
+}
+
+/* styling for the other pages - more will be added here */
```

Comment:
Creating sections like this is a great way to split up your CSS and be more organized. You could do this for each individual page